### PR TITLE
fix(owlbot): add owbot:run for trusted contributors

### LINKED ISSUE #1714: OwlBot checks should be made to be required checks
I think the problem is that we're not calling the github API to announce that the check was kicked off.  We only call the github API when the check has completed.  There's a window of time when the user isn't aware that the Owl Bot check is running, and may merge.

For example, this PR was merged before Owl Bot post processor completed.
https://github.com/googleapis/python-security-private-ca/pull/35

CC @parthea

### PR DESCRIPTION
**Pull request by bcoe**
Along with running `kokoro:run` for trusted contributors, we need to make sure to run `owlbot:run`.

Refs #1714

### CODE REVIEW

**Review by tmatsuo — CHANGES_REQUESTED** [review]
Please update the src/config-shcema.json as well.

**Review by tmatsuo — APPROVED** [review]

**Comment by tmatsuo** [comment]
The code itself looks good, but maybe we should use another type of label? `owlbot:run-from-fork`, or `owlbot:approve`?

**Comment by tmatsuo** [comment]
Per conversation on the chat.

Maybe we should add annotations only for PRs from a fork.

**Review by tmatsuo — CHANGES_REQUESTED** [review]
Can you also make the bot only add the annotations for PRs from a fork?

**Review by tmatsuo — APPROVED** [review]
Ok, I was wrong. We still needs annotations for PRs from the same repo.

File: packages/owl-bot/src/core.ts [hunk 1]
```diff
@@ -474,6 +474,32 @@ async function hasOwlBotLoop(
...
+ * @returns Promise was the last commit from OwlBot?
+ */
+async function lastCommitFromOwlBot(
+  owner: string,
+  repo: string,
+  prNumber: number,
+  octokit: Octokit
+): Promise<boolean> {
+  const commits = (
+    await octokit.pulls.listCommits({
```
Comment (tmatsuo) [inline packages/owl-bot/src/core.ts:493]: Probably it's alright, but there's an edge case where there are more than 100 commits.

**Review by tmatsuo — APPROVED** [review]

File: packages/owl-bot/src/core.ts (hunk 1 above)
Comment (bcoe) [reply packages/owl-bot/src/core.ts:493]: I don't think this API allows for more than 100 commits, maybe just add a comment.

Comment (bcoe) [reply packages/owl-bot/src/core.ts:493]: I'm wrong, it can potentially support pagination, I will add note.

**Review by tmatsuo — APPROVED** [review]

**Pull request merged**